ddtrace/llmobs/_llmobs.py

```diff
@@ -88,7 +99,29 @@ def _stop_service(self) -> None:
             log.warning("Failed to shutdown tracer", exc_info=True)
 
     @classmethod
-    def enable(cls, tracer=None):
+    def enable(
+        cls,
+        ml_app: Optional[str] = None,
+        integrations: Optional[List[str]] = None,
+        dd_llmobs_no_apm: bool = False,
+        dd_site: Optional[str] = None,
+        dd_api_key: Optional[str] = None,
+        dd_env: Optional[str] = None,
+        dd_service: Optional[str] = None,
```

Comment:
```suggestion
        llmobs_no_apm: bool = False,
        site: Optional[str] = None,
        api_key: Optional[str] = None,
        env: Optional[str] = None,
        service: Optional[str] = None,
```
Can we remove the `dd` prefixes? Just to keep things clean (happy to hear otherwise if you had reasons for adding the prefix)